Backend/app/rider-platform/rider-app/Main/src/API/UI/PartnerOrganizationMetro.hs

```diff
@@ -0,0 +1,130 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module API.UI.PartnerOrganizationMetro
+  ( API,
+    handler,
+  )
+where
+
+import qualified API.Types.UI.FRFSTicketService as DFRFSTypes
+import qualified Domain.Action.UI.FRFSTicketService as DFRFSTicketService
+import qualified Domain.Action.UI.PartnerOrganizationMetro as DPOM
+import qualified Domain.Types.FRFSTicketBooking as DFTB
+import qualified Domain.Types.PartnerOrgConfig as DPOC
+import Domain.Types.PartnerOrganization
+import qualified Domain.Types.Station as DStation
+import Environment
+import EulerHS.Prelude
+import Kernel.Beam.Functions as B
+import qualified Kernel.External.Maps as Maps
+import qualified Kernel.Storage.Esqueleto as DB
+import qualified Kernel.Storage.Hedis as Redis
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import Kernel.Utils.SlidingWindowLimiter (checkSlidingWindowLimitWithOptions)
+import Servant hiding (throwError)
+import Storage.Beam.SystemConfigs ()
+import qualified Storage.CachedQueries.Merchant.MerchantOperatingCity as CQMOC
+import qualified Storage.CachedQueries.PartnerOrgConfig as CQPOC
+import qualified Storage.CachedQueries.Station as CQS
+import Tools.Auth
+import Tools.Error
+
+type API =
+  "frfs"
+    :> "partnerOrganization"
+    :> ( PartnerOrganizationAPIKey
+           :> ( "upsertPersonAndGetFare"
+                  :> ReqBody '[JSON] DPOM.GetFareReq
+                  :> Post '[JSON] DPOM.GetFareResp
+                  :<|> "getConfig"
+                    :> ( "fromStation"
+                           :> Capture "fromGMMStationId" (Id DStation.GMMStation)
```

Comment:
we can make this generic by looking up partner org instead of hardcoding GMM in query param?